radar/languages_frameworks/micronaut.yml

```diff
@@ -0,0 +1,22 @@
+version: 1
+name: Micronaut
+blip:
+  - version: "2.6"
+    ring: TRIAL
```

Comment:
Assess or trial? Where is this used today?

Reply:
It is used in new Kafka Streams applications being developed for RS. We have not yet began running it in production, therefore it is not set to adopt. It seems very promising though, and gives many of the same abstractions as Spring Boot, while being a lot cheaper runtime-wise.